assets/javascript/test-data.js

```diff
@@ -0,0 +1,71 @@
+/******************************************************************************
+FSWD:  Christopher B. Zenner
+Date:  02/29/2020
+File:  them.js
+Ver.:  0.1.0 20200229
+This JS script provides the word data for the hangman-style game.
+******************************************************************************/
+var animals = {
```

Comment:
❓ Is this used anywhere? I see `dog-breeds` imported in your `index.js` but not `test-data`. If it's not used I would remove it to avoid confusion.